tests/Unit/DataStructures/DataBox/Test_DataBox.cpp

```diff
@@ -523,11 +525,24 @@ SPECTRE_TEST_CASE("Unit.DataStructures.DataBox.apply",
       CHECK(vector == (std::vector<double>{8.7, 93.2, 84.7}));
     }
   };
-  db::apply<
-      tmpl::list<test_databox_tags::Tag2, test_databox_tags::ComputeTag1>>(
-      ApplyCallable{}, original_box,
-      db::get<test_databox_tags::Tag1>(original_box));
+  db::apply(ApplyCallable{}, original_box,
+            db::get<test_databox_tags::Tag1>(original_box));
   /// [apply_struct_example]
+  /// [apply_stateless_struct_example]
+  struct ApplyStatelessCallable {
```

Comment:
Isn't this a `StatelessApplyCallable` not the application of a `StatelessCallable`?